conans/client/cmake.py

```diff
@@ -40,7 +40,8 @@ def _generator(self):
                         '10': '10 2010',
                         '11': '11 2012',
                         '12': '12 2013',
-                        '14': '14 2015'}
+                        '14': '14 2015',
+                        '15': '15 2017'}
```

Comment:
This part of the code is just related to the CMake generator that has to be used. Any implication of the toolset 14.1 that this code should take into account?